lib/active_merchant/billing/gateways/securion_pay.rb

```diff
@@ -1,9 +1,15 @@
 module ActiveMerchant #:nodoc:
   module Billing #:nodoc:
     class SecurionPayGateway < Gateway
+      class_attribute :shift4_v2_test_url
```

Comment:
This `class_attribute` is never used

Reply:
Thanks for the feedback! property excluded!